pandas/core/strings.py

```diff
@@ -2151,12 +2151,54 @@ def encode(self, encoding, errors="strict"):
         return self._wrap_result(result)
 
     _shared_docs['str_strip'] = ("""
-    Strip whitespace (including newlines) from each string in the
-    Series/Index from %(side)s. Equivalent to :meth:`str.%(method)s`.
+    Remove leading and trailing characters.
+
+    Strip whitespaces (including newlines) or a set of specified characters
+    from each string in the Series/Index from %(side)s.
+    Equivalent to :meth:`str.%(method)s`.
+
+    Parameters
+    ----------
+    to_strip : str
+        Specifying the set of characters to be removed. 
```

Comment:
Let's just simplify this and say "The set of characters to be removed". It matches the standard Python string method so no need to be too wordy